### PR TITLE
document climate.mqtt

### PR DESCRIPTION
**Pull request by clkao**
(This PR is superseded by #3448)

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#8750

### CODE REVIEW

File: source/_components/climate.mqtt.markdown [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+title: "MQTT HVAC"
+description: "Instructions how to integrate MQTT HVAC into Home Assistant."
+date: 2017-07-31 19:59
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: heat-control.png
+ha_category: Climate
+ha_release: 0.50
```
Comment (Landrash) [inline source/_components/climate.mqtt.markdown]: Release will need to be updated to 0.52.

File: source/_components/climate.mqtt.markdown [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+- **name** (*Required*): Name of MQTT HVAC. Default is 'MQTT HVAC'
+- **target_sensor** (*Required*): `entity_id` for a temperature sensor, target_sensor.state must be temperature.
+target temperature is 25 and the tolerance is 0.5 the heater will start when the sensor goes below 24.5 and it will stop when the sensor goes above 25.5.
+- **power_command_topic** (*Optional*): The MQTT topic to publish commands to change the power state. This is useful if your device has a separate power toggle in addition to mode.
+- **mode_command_topic** (*Required*): The MQTT topic to publish commands to change the HVAC operation mode state.
+- **temperature_command_topic** (*Required*): The MQTT topic to publish commands to change the target temperature.
+- **fan_mode_command_topic** (*Required*): The MQTT topic to publish commands to change the fan mode.
+- **swing_mode_command_topic** (*Required*): he MQTT topic to publish commands to change the swing mode.
+- **qos** (*Optional*): The maximum QoS level of the state topic. Default is 0 and will also be used to publishing messages.
```
Comment (Landrash) [inline source/_components/climate.mqtt.markdown]: Wrap the value in quotes for readability.

> `0`

File: source/_components/climate.mqtt.markdown [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+climate:
+  - platform: mqtt
+    name: Study
+    target_sensor: sensor.study_temperature
+```
+
+Configuration variables:
+
+- **name** (*Required*): Name of MQTT HVAC. Default is 'MQTT HVAC'
+- **target_sensor** (*Required*): `entity_id` for a temperature sensor, target_sensor.state must be temperature.
```
Comment (Landrash) [inline source/_components/climate.mqtt.markdown]: Where is tolerance specified?

File: source/_components/climate.mqtt.markdown [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+target temperature is 25 and the tolerance is 0.5 the heater will start when the sensor goes below 24.5 and it will stop when the sensor goes above 25.5.
+- **power_command_topic** (*Optional*): The MQTT topic to publish commands to change the power state. This is useful if your device has a separate power toggle in addition to mode.
+- **mode_command_topic** (*Required*): The MQTT topic to publish commands to change the HVAC operation mode state.
```
Comment (Landrash) [inline source/_components/climate.mqtt.markdown]: I would remove the word state at the end.

> - **mode_command_topic** (*Required*): The MQTT topic to publish commands to change the HVAC operation mode state.

> - **mode_command_topic** (*Required*): The MQTT topic to publish commands to change the HVAC operation mode.

File: source/_components/climate.mqtt.markdown [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+- **temperature_command_topic** (*Required*): The MQTT topic to publish commands to change the target temperature.
+- **fan_mode_command_topic** (*Required*): The MQTT topic to publish commands to change the fan mode.
+- **swing_mode_command_topic** (*Required*): he MQTT topic to publish commands to change the swing mode.
+- **qos** (*Optional*): The maximum QoS level of the state topic. Default is 0 and will also be used to publishing messages.
+- **retain** (*Optional*): If the published message should have the retain flag on or not.
+
+A full configuration example looks like the one below.
```
Comment (Landrash) [inline source/_components/climate.mqtt.markdown]: Add a header here to make a it easier to find and link to.

Example

``### {% linkable_title Example%}``
A full configuration example looks like the one below.

**Comment by clkao** [comment]
I've updated the PR except the release version, which depends on when home-assistant/home-assistant#8750 is ready.

**Pull request closed without merging**